packages/licenses/src/licenses.ts

```diff
@@ -0,0 +1,121 @@
+import { PnpmError } from '@pnpm/error'
+import {
+  Lockfile,
+} from '@pnpm/lockfile-file'
+import {
+  DependenciesField,
+  IncludedDependencies,
+  PackageManifest,
+  ProjectManifest,
+  Registries,
+} from '@pnpm/types'
+import { getPkgInfo } from './getPkgInfo'
+import { LicenseNode, lockfileToLicenseNodeTree } from './lockfileToLicenseNodeTree'
+
+export interface LicensePackage {
+  belongsTo: DependenciesField
+  version: string
+  packageManifest?: PackageManifest
+  packageName: string
+  license: string
+  licenseContents?: string
+  author?: string
+  packageDirectory?: string
```

Comment:
We usefully use a dir shortcat for directory in the repository
```suggestion
  packageDir?: string
```

Reply:
Updated the code to reflect this suggestion